pkg/pillar/types/zedroutertypes.go

```diff
@@ -1174,6 +1174,16 @@ type VifNameMac struct {
 	AppID   uuid.UUID
 }
 
+type AppNetworkInfo struct {
```

Comment:
yetus will require you to add a comment for this.
Might make sense to pick a more specific name such as AppNetworkAclArgs.

Reply:
sure, will do.